apps/src/p5lab/poetry/PoemSelector.jsx

```diff
@@ -7,19 +8,29 @@ import {APP_WIDTH} from '../constants';
 import {POEMS} from './constants';
 
 function PoemSelector(props) {
+  const [selectedPoem, setSelectedPoem] = useState(undefined);
+
   const onChange = e => {
     const poemTitle = e.target.value;
+    setSelectedPoem(poemTitle);
     const poem = Object.values(POEMS).find(poem => poem.title === poemTitle);
     if (poem) {
       props.onChangePoem(poem);
     }
   };
+
+  if (!selectedPoem) {
+    const defaultPoem = POEMS[appOptions.level.defaultPoem];
```

Comment:
Assumes that `level.defaultPoem` will always be set and will always be a valid poem. I think this will always based on the way the levelbuilder field works (it's impossible to leave blank, and all of the values match poems in the `POEMS` constant on the frontend).